test/e2e/libpod_suite_remote_test.go

```diff
@@ -138,6 +139,11 @@ func (p *PodmanTestIntegration) StopRemoteService() {
 	if err := os.Remove(socket); err != nil {
 		fmt.Println(err)
 	}
+	if p.RemoteSocketLock != "" {
+		if err := os.Remove(p.RemoteSocketLock); err != nil {
+			fmt.Println(err)
```

Comment:
This is a useful thing to print, but should it be using `logrus.Warnf()` like all the others?

Reply:
It’s consistent with the other failure, and nothing else in this function uses `logrus`.

Changing that might well be the right thing to do, but feels a bit out of scope.

Reply:
okay, you know more than me.

Reply:
(I definitely don’t, this is the first time I’ve seen this code and I have no idea what Ginkgo likes.)